app-misc/pfetch/pfetch-0.1.recipe

```diff
@@ -0,0 +1,29 @@
+SUMMARY="A pretty system information tool written in POSIX sh"
+DESCRIPTION="The goal of this project is to implement a simple system \
+information tool in POSIX sh using features built into the language itself \
+(*where possible*). The source code is highly documented and the author hopes \
+it will act as a learning resource for POSIX sh and simple information \
+detection across various different operating systems."
+HOMEPAGE="https://github.com/dylanaraps/pfetch"
+COPYRIGHT="2016-2019 Dylan Araps"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/$portVersion/$portVersionedName.tar.gz"
+CHECKSUM_SHA256="0f42383ce099c7a3956fe93c803b4882c094e869d4dd09b22806e83b176abe2d"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	pfetch = $portVersion
+	cmd:pfetch
+	"
+REQUIRES="
+	haiku
+	"
+
+INSTALL()
+{
+	install -d "$binDir" "$docDir"
+	install -m 755 pfetch "$binDir"
+	install -m 644 README.md "$docDir"
```

Comment:
Maybe install in $docDir/pfetch?

Reply:
Doesn't `$docDir` already include the port name in the path?

Reply:
My bad, didn't check the recipe as I think it looks ok, if the subfolder is created it's ok as it is :)